packages/react-native/React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputComponentView.mm

```diff
@@ -555,12 +553,9 @@ - (void)blur
 #if !TARGET_OS_OSX // [macOS]
   [_backedTextInputView resignFirstResponder];
 #else // [macOS
-  NSWindow *window = _backedTextInputView.window;
-  if (window && window.firstResponder == _backedTextInputView) {
-    // Calling makeFirstResponder with nil will call resignFirstResponder and make the window the first responder
-    [window makeFirstResponder:nil];
-  }
-#endif // macOS];
+  NSWindow *window = [_backedTextInputView window];
+  [window makeFirstResponder:nil];
```

Comment:
This check should still be done. The window might be defined, but the component might not be the first responder. Without this check, you may blur another view

'''
window && window.firstResponder == _backedTextInputView
'''

Reply:
Ah thanks I totally missed that. I'll bring that back.